### DISCUSSION TITLE
Unable to create an app using the Esty API

### DISCUSSION THREAD

**Discussion by Thangduong24nd** [etsy/open-api#1146] [Q&A]
Hi, I'm applying for an app using Etsy API for me to use only to manage my own shop, not for commercial use. In the website URL field, what should I enter? Should I enter my Etsy shop URL?

Thank you!
![image](https://github.com/etsy/open-api/assets/73693556/db0f4638-7715-4ce9-a105-06760aaf7d9f)

**Comment by leeihuayu** [comment]
I also want to know this question because I don't have a website and just want to write a desktop program.
